tests/stress/benchmarkstress_test.go

```diff
@@ -0,0 +1,90 @@
+package system
+
+import (
+	"strconv"
+	"testing"
+)
+
+const (
+	deployCommand = "while true ; do echo LONG RUNNING SERVICE ; sleep 5 ; done"
```

Comment:
Can you explain how this works and why it's needed?

Reply:
^^^ disregard my question as long as you remove `deployCommand` as a param for `benchmarkStress`, in which case this makes a lot more sense

Reply:
✅